test/gtest/ucp/test_ucp_ep_reconfig.cc

```diff
@@ -327,12 +423,13 @@ UCS_TEST_P(test_ucp_ep_reconfig, request_reset, "PROTO_REQUEST_RESET=y")
     run();
 }
 
-/* SHM causes lane reuse, disable for now. */
-UCS_TEST_SKIP_COND_P(test_ucp_ep_reconfig, resolve_remote_id,
-                     has_transport("shm") || is_self(), "MAX_RNDV_LANES=0")
+UCS_TEST_SKIP_COND_P(test_ucp_ep_reconfig, resolve_remote_id, is_self(),
+                     "MAX_RNDV_LANES=0")
 {
     if (has_transport("tcp")) {
-        UCS_TEST_SKIP_R("lanes are reused (not supported yet)");
+        UCS_TEST_SKIP_R("asymmetric setup is not supported for this transport "
+                        "due to a reachability issue - only matching "

```

Comment:
why TCP is considered "not matching" interface?

Reply:
TCP ifaces use the same MD, so they are identified as a single remote resource in lane selection, which disrupts reconfig process.